packages/wdio-browserstack-service/src/Percy/PercySDK.ts

```diff
@@ -17,9 +19,19 @@ let snapshotHandler = (...args: any[]) => {
     PercyLogger.error('Unsupported driver for percy')
 }
 if (percySnapshot) {
-    snapshotHandler = (browser: WebdriverIO.Browser | WebdriverIO.MultiRemoteBrowser, name: string) => {
+    snapshotHandler = (browser: WebdriverIO.Browser | WebdriverIO.MultiRemoteBrowser, snapshotName: string, options?: any) => {
         if (process.env.PERCY_SNAPSHOT === 'true') {
-            return percySnapshot(browser, name)
+            let { name, uuid } = InsightsHandler._currentTest
+            if (!name || name === '') {
+                ({ name, uuid } = TestReporter._currentTest)
```

Comment:
Why surrounding parenthesis ?

Reply:
Destructured assignment for declared variables under a statement block needs parentheses in typescript.